src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -69,132 +76,147 @@ Release properties overlap with event and issue search properties, so they don't
+| spans.db | Cumulative db time for a transaction, based on span operations. | x |  |  | duration |
+| spans.http | Cumulative http time for a transaction, based on span operations. | x |  |  | duration |
+| spans.resource | Cumulative resource time for a transaction, based on span operations. | x |  |  | duration |
+| stack.abs_path | The absolute path to the source file. In events, this is an array; in issues, this is a single value. | x | x |  | array, single value |
+| stack.colno | Column number of the call, starting at 1. | x |  |  | array |
+| stack.filename | The path to the source file relative to the project root directory. In events, this is an array. In issues, this is a single value. | x | x |  | array, single value |
+| stack.function | Name of the function being called. In events, this is an array. In issues, this is a single value. | x | x |  | array, single value |
+| stack.in_app | Indicates whether a frame is related to the execution of the relevant code in the stack trace. For example, the frames that might power the framework’s web server of your app are probably not relevant. However, calls to the framework’s library once you start handling code likely are relevant. Values can be `1` (true) or `0` (false). | x |  |  | array |
+| stack.lineno | Line number of the call, starting at 1. | x |  |  | array |
+| stack.module | Platform-specific module path. For example, `sentry.interfaces.Stacktrace`. In events, this is an array. In issues, this is a single value. | x | x |  | array, single value |
+| stack.package | The "package" the frame was contained in. Depending on the platform, this can be different things. For C#, it can be the name of the assembly. For native code, it can be the path of the dynamic library or something else. In events, this is an array. In issues, this is a single value. | x | x |  | array, single value |
+| sum(numeric field) | Returns results with a matching total value for the the field entered. | x |  |  | matches field |
+| timesSeen | Returns results with a matching count. (Same as `count()` in events.) |  | x |  | number |
+| timestamp | The finish timestamp of the transaction. Returns events with matching datetime. | x | x |  | datetime |
+| timestamp.to_day | Timestamp rounded down to the nearest day. | x |  |  | datetime |
+| timestamp.to_hour | Timestamp rounded down to the nearest hour. | x |  |  | datetime |
+| title | Title of the error or the transaction name. | x | x |  | string |
+| trace | A trace represents the record of the entire operation you want to measure or track — like page load, searched using the uuid generated by Sentry’s SDK. | x | x |  | uuid |
+| trace.parent_span | Span ID of the parent to the current transaction. This is null if the transaction is root. | x |  |  | uuid |
+| trace.span | Span ID of the root span of the root transaction in the event. | x |  |  | uuid |
```

Comment:
```suggestion
| trace | A trace represents the record of the entire operation you want to measure or track — like page load, searched using the uuid generated by Sentry’s SDK. | x | x |  | uuid |
| trace.parent_span | Span ID of the parent to the current transaction. This is null if the transaction is root. | x |  |  | uuid |
| trace.span | Span ID of the root span of the root transaction in the event. | x |  |  | uuid |
| traceIds| A specific trace id that the user created during the replay. |  |  | x | string |
```

Reply:
LGTM

Reply:
we're combining `traceIds` with the existing `trace` field.

Reply:
Thanks for the heads up!